packages/notifications/src/styled/content/StyledClose.tsx

```diff
@@ -5,30 +5,51 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
+import React, { HTMLAttributes } from 'react';
 import styled from 'styled-components';
-import classNames from 'classnames';
-import CalloutStyles from '@zendeskgarden/css-callouts';
 import { retrieveComponentStyles } from '@zendeskgarden/react-theming';
+import XStrokeIcon from '@zendeskgarden/svg-icons/src/12/x-stroke.svg';
+import { IStyledAlertProps } from '../StyledAlert';
 
 const COMPONENT_ID = 'notifications.close';
 
-export interface IStyledCloseProps {
+export interface IStyledCloseProps extends IStyledAlertProps {
   isFocused?: boolean;
   isHovered?: boolean;
 }
 
+const Button = React.forwardRef<HTMLButtonElement, HTMLAttributes<HTMLButtonElement>>(
+  (props, ref) => (
+    <button ref={ref} {...props}>
+      <XStrokeIcon />
+    </button>
+  )
+);
+
 /**
  * Used to close a Notification. Supports all `<button>` props
+ *
+ * 1. Reset for <button> element.
  */
-export const StyledClose = styled.button.attrs<IStyledCloseProps>(props => ({
+export const StyledClose = styled(Button).attrs({
   'data-garden-id': COMPONENT_ID,
-  'data-garden-version': PACKAGE_VERSION,
-  type: 'button',
-  className: classNames(CalloutStyles['c-callout__close'], {
-    // State
-    [CalloutStyles['is-focused']]: props.isFocused,
-    [CalloutStyles['is-hovered']]: props.isHovered
-  })
-}))<IStyledCloseProps>`
+  'data-garden-version': PACKAGE_VERSION
+})<IStyledCloseProps>`
+  display: block;
+  position: absolute;
+  top: ${props => props.theme.space.xxs};
+  right: ${props => !props.theme.rtl && props.theme.space.xxs};
+  left: ${props => props.theme.rtl && props.theme.space.xxs};
```

Comment:
> For starters, please go through the entire PR and replace all `space.[dimension]` sizing with the corresponding `space.base` calculations. We want to pack a lot of leverage into a user's ability to modify the single `space.base` theme variable. The `xs`, `sm`, etc. sizing dimensions are generally available for convenience and used solely for `grid`.

It seems the `space` sizing is [internally computed](https://github.com/zendeskgarden/react-components/blob/next/packages/theming/src/elements/theme/index.ts#L133) using `space.base`, so these values may not need to be changed. There are two theme values that are used in this PR that isn’t computed internally with `space.base`: 

1) `border`
2) `fontSizes` 

I can replace these values with `space.base` calculations. Is there a place that documents which properties are computed using `space.base`? For example, as the end-user, if I modify the theme's `base` value, what parts of the design system will change its size?

Reply:
Update: after speaking with @jzempel, I learned that the computed values happen before the user applies a custom theme. I'll update `theme.space` usages in this PR to be computed with `space.base` 👍 .